2024-07-31 18:20:01,035 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-07-31 18:20:01,037 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\XY262ED\OneDrive - EY\Desktop\Django Projects\SMT_6\SMT\.venv\Lib\site-packages\instabot\bot\bot.py
2024-07-31 18:20:01,557 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-07-31 18:20:01,705 - instabot version: 0.117.0 (api) - WARNING - HTTPSConnectionPool(host='i.instagram.com', port=443): Max retries exceeded with url: /api/v1/accounts/get_prefill_candidates/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')))
2024-07-31 18:20:01,769 - instabot version: 0.117.0 (api) - WARNING - HTTPSConnectionPool(host='i.instagram.com', port=443): Max retries exceeded with url: /api/v1/qe/sync/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')))
2024-07-31 18:20:01,829 - instabot version: 0.117.0 (api) - WARNING - HTTPSConnectionPool(host='i.instagram.com', port=443): Max retries exceeded with url: /api/v1/launcher/sync/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')))
2024-07-31 18:20:01,888 - instabot version: 0.117.0 (api) - WARNING - HTTPSConnectionPool(host='i.instagram.com', port=443): Max retries exceeded with url: /api/v1/accounts/contact_point_prefill/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')))
2024-07-31 18:20:01,891 - instabot version: 0.117.0 (bot) - INFO - Bot stopped. Worked: 0:00:00.863263
2024-07-31 18:20:01,892 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: C:\Users\XY262ED\OneDrive - EY\Desktop\Django Projects\SMT_6\SMT\Tool/config/smtdemoteam.checkpoint
2024-07-31 18:20:01,893 - instabot version: 0.117.0 (bot) - INFO - Total requests: 4
